content/en/cloudcraft/getting-started/group-by-presets.md

```diff
@@ -0,0 +1,78 @@
+
+In Azure, you can group resources by:
+- Resource Group
+- Region
+- VNet
+- Subnet
+
+## Presets
+
+**Presets** offer a convenient way to apply predefined sets of group-bys and filters, allowing you to quickly view your resources from different perspectives. This feature simplifies the process of applying groupings and filters to your diagrams, allowing you to focus on specific aspects of your architecture.
+
+**Cloudcraft provides three built-in presets:** Infrastructure, Network, and Security. These presets are designed to address different operational needs.
+
+{{< img src="cloudcraft/getting-started/group-by-presets/diagram-presets.png" alt="Cloudcraft interface showing preset options with the infrastructure diagram preset selected." responsive="true" style="width:100%;">}}
+
+To apply a preset to your diagram:
+
+1. Switch to the **Live** tab within Cloudcraft.
+2. Select the desired preset from the menu at the top of the diagram view.
+3. The diagram automatically updates to reflect the chosen preset.
+
+### Infrastructure diagram
+
+{{< img src="cloudcraft/getting-started/group-by-presets/infrastructure-diagram.png" alt="Infrastructure diagram showing servers, databases, security components, and the relationship between them." responsive="true" style="width:100%;">}}
+
+The infrastructure preset provides a broad overview, grouping resources by Region and VPC in AWS, and by Region and VNet in Azure. This preset is ideal for quickly generating architecture diagrams for troubleshooting or high-level review.
+
+- In AWS, it deactivates components like EBS, NAT Gateway, and Transit Gateway, among others.
+- In Azure, components such as Azure VNGW and Azure Disk are disabled.
+
+### Security diagram
+
+{{< img src="cloudcraft/getting-started/group-by-presets/security-diagram.png" alt="Security diagram showing servers, databases, security components, and the relationship between them." responsive="true" style="width:100%;">}}
+
+The security preset focuses on potential security exposures, grouping resources by Region, VPC, and Security Group in AWS. This view is essential for identifying security risks and understanding rules governing inbound and outbound service communications, and is perfect for mapping attack surfaces during penetration testing or security audits.
+
+- This preset does not currently support Azure configurations.
+- In AWS, similar to infrastructure, it disables EBS, NAT Gateway, and other components that might clutter the security view.
+
+### Network diagram
+
+{{< img src="cloudcraft/getting-started/group-by-presets/network-diagram.png" alt="Network diagram showing servers, databases, security components, and the relationship between them." responsive="true" style="width:100%;">}}
+
+The Network preset adds granularity by introducing Subnet grouping, making it especially useful for network teams seeking to identify latency sources and traffic patterns.
+
+- In AWS, it disables components such as EBS, S3, and SNS.
```

Comment:
ame comment - rephrase to not say disabled. Alternatively, you can make it generic saying the view only shows components that highlight the networking. It might also be worth while explaining that the components may appear multiple times if they are part of subnets